src/js/server/apply/router.js

```diff
@@ -1,9 +1,19 @@
 const express = require('express');
 const { createApplicationForm } = require('js/shared/application-form');
 const renderForm = require('js/shared/render-form');
+var querystring = require('querystring');
+var fetch = require('node-fetch');
+const aws = require('aws-sdk');
+const multer = require('multer');
+const multerS3 = require('multer-s3');
+const crypto = require('crypto');
```

Comment:
It looks like you got some of these back with a merge somewhere. Could you remove these dependencies?